specs/notation-config.md

```diff
@@ -0,0 +1,63 @@
+# Notation Configuration
+
+To enable persisted configuration, simplifying the execution of the `notation` cli, the following configuration file will be available
+
+> Note: there will be a policy based configuration that will come at a later point.
+
+## Location
+
+The default location and file will be stored at: `~/.notary/notation-config.json`. The `notation` cli and libraries will support alternate locations through a `config-location` parameters.
+
+> TODO: Add Windows and Mac locations
+
+## Properties
+
+Property | Type |  Value
+------ | ------ | ---
+`enabled`|_bool_|on/off switch for default notation behavior. --enabled false would disable all automatic validations
```

Comment:
`enabled` is used for `docker-notation`. This field is not meaningful for the `notation` CLI.

Reply:
@shizhMSFT, this is in the current config file. Are we having separate configs for docker-notation and notation?